lib/middleware.js

```diff
@@ -512,6 +516,22 @@ module.exports = function middlewareConstructor(env) {
           res.json({"status": 200});
         });
       };
+    },
+
+    /**
+     * If there's an oauth token, decrypt it and set the
+     * local user object.
+     */
+    maybeDecryptToken: function(req, res, next) {
```

Comment:
It's something I've seen in a number of places for when a function might perform an action. Can you think of a better name?